frontend/src/components/requisitionForm/RequisitionItemCard.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react";
+import { Button, Card, Col, Form, Input, InputNumber, Row, Tooltip } from "antd";
+import { QuestionCircleOutlined } from "@ant-design/icons/lib";
+
+const { TextArea } = Input;
+
+interface FieldData {
+  name: number;
+  key: number;
+  fieldKey: number;
+}
+
+interface Props {
+  field: FieldData;
+  deleteButton: boolean;
+  rules: any;
+  remove: (index: number) => void;
+}
+
+const RequisitionItemCard: React.FC<Props> = (props) => (
+  <Card
+    size="small"
+    title={`Item ${props.field.name + 1}`}
+    style={{ marginBottom: "15px" }}
+    extra={props.deleteButton
+      ? (
+        <Button onClick={() => props.remove(props.field.name)} type="text" size="small" danger>
+          Delete
+        </Button>
+      ) : null}
+  >
+    <Row gutter={[8, 0]}>
+      <Col xs={24} sm={12}>
+        <Form.Item
+          name={[props.field.name, "name"]}
+          fieldKey={[props.field.fieldKey, "name"]}
+          rules={[props.rules.requiredRule]}
+          label="Name"
+        >
+          <Input placeholder="Sparkly crayons" />
+        </Form.Item>
+      </Col>
+
+      <Col xs={24} sm={12}>
+        <Form.Item
+          name={[props.field.name, "link"]}
+          fieldKey={[props.field.fieldKey, "link"]}
+          rules={[props.rules.requiredRule, props.rules.urlRule]}
+          label={(
+            <span>
+              {"Link "}
+              <Tooltip title="The link to the item you want to purchase">
+                <QuestionCircleOutlined />
+              </Tooltip>
+            </span>
+          )}
+        >
+          <Input type="url" placeholder="http://rubberducks.com" />
```

Comment:
this is an actual website that redirects to http://game-fundraising.com/ 😂

Reply:
Haha that's funny